frontend/packages/dev-console/integration-tests/features/user-preferences/user-preferences-dev-perspective.feature

```diff
@@ -0,0 +1,92 @@
+@odc-5227
+Feature: Manage user preferences
+              As a user, I need the ability to set/edit my preferences for the OCP Console.
+
+
+        Background:
+            Given user is at developer perspective
+
+
+        @regression @to-do
+        Scenario: Setting Developer preference for perspective: UP-01-TC01
+            Given user in at admin perspective
+             When user clicks on user dropdown on masthead
+              And user clicks User Preference option
+              And user clicks on Perspective dropdown on User Preferences page
+              And user selects "Developer" option
+              And user clicks on Red Hat OpenShift Container Platform logo on masthead
+             Then user can see "Developer" option selected in the perspective switcher
+
+
+        @regression @to-do
+        Scenario: Setting a preference for a project: UP-01-TC02
+            Given user has created project "test-preference1"
+             When user clicks on user dropdown on masthead
+              And user clicks User Preference option
+              And user clicks on Project dropdown on User Preferences page
+              And user searches project "test-preference1"
+              And user selects the project "test-preference1" from the dropdown
+              And user clicks on Add present in navigation menu
+             Then user can see "test-preference1" option selected in project namespace
```

Comment:
I miss here the additional test that, if the user opens the console that the selected project is preselected.

```suggestion
             Then user can see "test-preference1" option selected in project namespace
              And user switches to the add page the project "test-preference1" is preselected.
```

Reply:
Updated